analysis/design.R

```diff
@@ -92,6 +92,11 @@ events_lookup <- tribble(
   "coviddeath", "coviddeath_date", "COVID-19 death",
   "noncoviddeath", "noncoviddeath_date", "Non-COVID-19 death",
   "death", "death_date", "Any death",
+  "fracture", "fracture_date", "Fracture",
+  "fractureemergency","fractureemergency_date","Fracture A&E attendance",
+  "fractureadmitted","fractureadmitted_date","Fracture hospitalisation",
+  "fracturedeath","fracturedeath_date","Fracture death",
```

Comment:
Don't think we need these as we're only interested in any fracture as an outcome, not where it showed up in the health record. But no harm (i think) to leave in.

Reply:
It probably makes very little practical difference but It is being used to select variables in [process_controlactual.R](https://github.com/opensafely/vaccine-effectiveness-in-kids/blob/fractures/analysis/matching/process_controlactual.R#L211) so I will keep it in for now.